index.bs

```diff
@@ -1568,6 +1580,54 @@ for accelerometer sensor is given below.
     };
 </pre>
 
+<h3 id="feature-policy-api">Extending the Feature Policy API</h3>
+
+An implementation of the {{Sensor}} interface for each [=sensor type=] has a
+[=policy-controlled feature=] that controls whether or not this implementation
+can be used in a document.
+
+The [=policy-controlled feature|feature=]'s [=default allowlist=] is
+<code>["self"]</code>.
+
+Note: The [=default allowlist=] of <code>["self"]</code> allows {{Sensor}} interface
+implementation usage in same-origin nested frames but prevents third-party content
+from [=sensor readings=] access.
+
+The [=feature name=] for this [=policy-controlled feature|feature=]
+is [=sensor feature name=].
+
+A [=Low-level=] {{Sensor|sensor}} may use its interface name as a [=sensor feature name=],
+for instance, "gyroscope" or "accelerometer". Unless the [=extension specification=] tells
+otherwise, the [=sensor feature name=] matches the [=sensor type|type=]-associated
+{{PermissionName}}.
+
+<div class="example html">
+Third-party Accelerometer usage is selectively enabled by adding [=allow attribute=]
+to the frame container element:
+<pre highlight="html">
+  &lt;iframe src="https://third-party.com" allow="accelerometer"/&gt;&lt;/iframe&gt;
+</pre>
+</div>
+
+<div class="example html">
+A sensor usage is disabled completely by specifying the feature policy in a HTTP
+response header:
+<pre highlight="js">
+ Feature-Policy: {"accelerometer": []}
+</pre>
+</div>
+
+[=sensor fusion|Fusion sensors=] must use [=sensor feature names=] of the sensors
+that are used as a source of fusion.
+
+<div class="example html">
+Enabling third-party usage for an absolute orientation sensor:
```

Comment:
This might be misleading, maybe:

Allow third-party origin to use accelerometer, magnetometer and gyroscope features that are required by the absolute orientation sensor.